src/wal.rs

```diff
@@ -170,6 +172,11 @@ impl Wal {
     }
 
     /// Encode entry to buffer
+    ///
+    /// The entry is encoded to a header followed by plain key and value.
```

Comment:
What I want is like this.
<img width="342" alt="截屏2020-12-09 上午11 18 59" src="https://user-images.githubusercontent.com/13497871/101569986-757dd280-3a10-11eb-87b1-93dbae179e63.png">

"Header consists of ...." should be included in the comment of header encode() instead of here.

Reply:
Alright, I'll add this.

Reply:
Header format and WAL format has been added.